symphony/bdk/core/service/health/health_service.py

```diff
@@ -0,0 +1,40 @@
+from symphony.bdk.gen.agent_api.system_api import SystemApi
+from symphony.bdk.gen.agent_api.signals_api import SignalsApi
+
+from symphony.bdk.gen.agent_model.v3_health import V3Health
+
+
+class HealthService:
+    """Service class for checking health of the Agent server."""
+
+    def __init__(self, system_api: SystemApi, signals_api: SignalsApi):
+        self._system_api = system_api
+        self._signals_api = signals_api
+
+    async def health_check(self) -> V3Health:
+        """ Returns the connectivity status of your Agent server.
+        If your Agent server is started and running properly, the status value will be UP.
+        Available on Agent 2.57.0 and above.
+
+        :return: V3Health: the connectivity status of your Agent server.
+        `Health Check v3 <https://developers.symphony.com/restapi/reference#health-check-v3>`_
+        """
+        return await self._system_api.v3_health()
+
+    async def health_check_extended(self) -> V3Health:
+        """Returns the connectivity status of the Agent services as well as users connectivity.
+        Available on Agent 2.57.0 and above.
+
+        :return: V3Health: the connectivity status of the Agent services as well as users connectivity.
+        `Healt Check Extended v3 <https://developers.symphony.com/restapi/reference#health-check-extended-v3>`_
```

Comment:
It is better to put the links to the documentation before the `:return:` statement IMHO